services/front/vite.config.ts

```diff
@@ -6,11 +6,31 @@ import checker from "vite-plugin-checker";
 import tsconfigPaths from "vite-tsconfig-paths";
 import { VitePWA } from "vite-plugin-pwa";
 import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
-
 export default defineConfig({
     plugins: [
         react(),
-        VitePWA({ registerType: "autoUpdate", selfDestroying: true }),
+        VitePWA({
+            strategies: 'injectManifest', injectManifest: {
+                maximumFileSizeToCacheInBytes: 10000000
```

Comment:
is that how you solved the max file chunk issue?

Reply:
?

Reply:
yes